### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by SyedSibtainRazvi**
# Related Issue
https://github.com/code-mancers/gather-bot/issues/55

# functionalities Added or Changed

- Added GitHub actions to build and test in Node

# Additional Info

- Every time a PR is opened, test cases should run on the PR
Use Postgres for running test cases
PR will be reviewed once the test cases are passed

# Checklist

- [ ] Tests
- [ ] Uploaded screenshots

# Screenshots

### CODE REVIEW

File: .github/workflows/g-bot-actions.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      
+      - name: Install dependencies
+      - run: npm ci
+
+       - name: Connect to PostgreSQL
+        run: node client.js
```
Comment (SujayPrabhu96) [inline .github/workflows/g-bot-actions.yml:49]: Do we have client.js file in our file ?

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: .github/workflows/g-bot-actions.yml (hunk 1 above)
Comment (SyedSibtainRazvi) [reply .github/workflows/g-bot-actions.yml:40]: No, removing it

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]
We need to specify the database
Reference: https://gist.github.com/2color/537f8ef13ecec80059abb007839a6878

**Comment by SujayPrabhu96** [comment]
> > We need to specify the database Reference: https://gist.github.com/2color/537f8ef13ecec80059abb007839a6878
> 
> I have a question here. I saw the reference and we need to specify the database, however why localhost DB? And since we are using Prisma then we need to add that as well.

No, its a reference. You can get github action to set up Postgres in Marketplace

File: .github/workflows/g-bot-actions.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:5
```
Comment (SujayPrabhu96) [inline .github/workflows/g-bot-actions.yml:31]: Why 5 ?

Comment (SyedSibtainRazvi) [reply .github/workflows/g-bot-actions.yml]: Removed it

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**